composer.json

```diff
@@ -27,45 +26,42 @@
         }
     ],
     "require": {
-        "php": "^5.6 || ^7.0",
-        "a2lix/auto-form-bundle": "@dev",
-        "symfony/config": "^2.8 || ^3.4 || ^4.0",
-        "symfony/dependency-injection": "^2.8 || ^3.4 || ^4.0",
-        "symfony/doctrine-bridge": "^2.8 || ^3.4 || ^4.0",
-        "symfony/event-dispatcher": "^2.8 || ^3.4 || ^4.0",
-        "symfony/form": "^2.8 || ^3.4 || ^4.0",
-        "symfony/http-foundation": "^2.8 || ^3.4 || ^4.0",
-        "symfony/http-kernel": "^2.8 || ^3.4 || ^4.0",
-        "symfony/options-resolver": "^2.8 || ^3.4 || ^4.0"
+        "php": "^7.1.3",
+        "a2lix/auto-form-bundle": "0.1-beta.3",
+        "symfony/config": "^3.4|^4.0",
+        "symfony/dependency-injection": "^3.4|^4.0",
+        "symfony/doctrine-bridge": "^3.4|^4.0",
+        "symfony/event-dispatcher": "^3.4|^4.0",
+        "symfony/form": "^3.4|^4.0",
+        "symfony/http-foundation": "^3.4|^4.0",
+        "symfony/http-kernel": "^3.4|^4.0",
+        "symfony/options-resolver": "^3.4|^4.0"
     },
     "require-dev": {
-        "doctrine/common": "^2.7",
-        "doctrine/doctrine-bundle": "^1.8",
-        "doctrine/orm": "^2.5",
-        "gedmo/doctrine-extensions": "^2.4",
```

Comment:
Does this mean it does not support gedmo anymore?

Reply:
cc @webda2l

Reply:
I didn't try since a long time but I think it's always compatible with gedmo personal translation strategy (http://atlantic18.github.io/DoctrineExtensions/doc/translatable.html#personal-translations).

With default gedmo strategy (http://atlantic18.github.io/DoctrineExtensions/doc/translatable.html#entity-domain-object), only deprecated 1.x version of TranslationFormBundle should continue to handle it with dedicated form type, see https://github.com/a2lix/TranslationFormBundle/blob/1.x/README.md

Reply:
Thank you, will try to find some time and test it.

Reply:
You're welcome. Yes give the feedback of your test please.